README.md

```diff
@@ -58,31 +58,28 @@ If you encounter a problem, have some genius, crazy idea or just want to have a
 
 ### First steps
  
- 1. Download and install the [Arduino IDE](https://www.arduino.cc/en/Main/Software)
- 2. Download and add the [ESP8266 hardware extension to Arduino](https://github.com/esp8266/Arduino). **If you already have Arduino IDE and the ESP8266 Hardware extension, make sure you upgrade these to the latest version.**
- 3. Download and add the COOL Board library set to the Arduino IDE
-     1. First Method
-         * Open the Arduino IDE
-         * Go to the `Sketch` Menu 
-         * Choose `Include Library > Manage Libraries.` 
-         * Search for `CoolBoard`
-         * Install
-         * Restart the Arduino IDE
-     2. Second Method (if you have a CoolBoard.zip file )
-         * Open the Arduino IDE
-         * Go to the `Sketch > Include Library > Add .ZIP Library` menu item
-         * Search for `CoolBoard.zip`
-         * Click Open
-         * Restart the Arduino IDE
-     3. Third Method (with the library GitHub link)
-         * Open Arduino IDE
-         * Go to the `File > Preferences` menu item
-         * Check "SketchBook Location" path
-         * Go to the `Arduino/libraries` folder (create it if necessary)
-         * Clone the repository the there (`git clone <github_repo_url`)
-         * Restart the Arduino IDE
-4. Download [ESP8266 SPIFFS Tool](https://arduino-esp8266.readthedocs.io/en/latest/filesystem.html#uploading-files-to-file-system)
-5. **Optional but highly recommended:** Download the [ESP8266 Exception Decoder](https://github.com/me-no-dev/EspExceptionDecoder/releases/tag/1.0.6) and follow its [installation guide](https://github.com/me-no-dev/EspExceptionDecoder)
+ 1. Download and install the Visual Studio Code (https://code.visualstudio.com/)
```

Comment:
"Install PlatformIO (using either the Atom based distribution, the Visual Studio code plugin or platform IO core)"

Provide links for each possible alternative.